app/src/main/java/com/keylesspalace/tusky/entity/Card.kt

```diff
@@ -13,21 +13,30 @@
  * You should have received a copy of the GNU General Public License along with Tusky; if not,
  * see <http://www.gnu.org/licenses>. */
 
-package com.keylesspalace.tusky.entity;
+package com.keylesspalace.tusky.entity
+
+import android.os.Parcelable
+import kotlinx.android.parcel.Parcelize
+
+@Parcelize
+data class Card (val url: String,
```

Comment:
Fields formatting? See 'Class header formatting'
https://kotlinlang.org/docs/reference/coding-conventions.html
"Classes with longer headers should be formatted so that each primary constructor parameter is in a separate line with indentation. Also, the closing parenthesis should be on a new line. If we use inheritance, then the superclass constructor call or list of implemented interfaces should be located on the same line as the parenthesis:"